libs/geography/src/selectors/geography.selector.ts

```diff
@@ -36,7 +36,7 @@ const createGeographySelectors = <T extends DaffCountry>() => {
   }
 }
 
-export const daffGeographySelectors = (() => {
+export const getDaffGeographySelectors = (() => {
```

Comment:
I think this should be named `getDaffGeographyPageSelectors`. That way the base name can be reserved for the exposed named.

Reply:
Actually, hold up. Maybe that name doesn't make sense in this case

Reply:
maybe `getGeographySelectors`, since it won't be exposed to the end user? I'm just trying to reserve the neutral name for the all-selector export.

Reply:
yeah seems good, changed